view/sharedcache/core/SharedCache.cpp

```diff
@@ -2990,7 +2998,7 @@ bool SharedCache::SaveToDSCView()
 		c.m_dyldDataRegions = m_dyldDataRegions;
 		c.m_nonImageRegions = m_nonImageRegions;
 		c.m_baseFilePath = m_baseFilePath;
-		c.m_exportInfos = m_exportInfos;
+		//c.m_exportInfos = m_exportInfos;
```

Comment:
I guess you commented this out while testing the `std::unique_ptr` approach.

Reply:
Yup 🤦